### PR TITLE
Add privilege for media-controller

### PR DESCRIPTION
**Pull request by haejeongkim**
Add description for privilege for media-controller

### CODE REVIEW

File: docs/application/native/guides/multimedia/media-controller.md [hunk 1]
```diff
@@ -107,13 +113,20 @@ To enable your application to use the media controller functionality:
 
 - To use the media controller client:
 
-  1. To use the functions and data types of the Media Controller Client API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) applications), include the `<media_controller_client.h>` header file in your application:
+  1. The application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```
Comment (mijin85cho) [inline docs/application/native/guides/multimedia/media-controller.md]: Would you please update like below?
```
1. To use the Media Controller Client API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

File: docs/application/native/guides/multimedia/media-controller.md [hunk 2]
```diff
@@ -107,13 +113,20 @@ To enable your application to use the media controller functionality:
...
+     ```
+     <privileges>
+        <privilege>http://tizen.org/privilege/mediacontroller.client</privilege>
+     </privileges>
+     ```
+  
+  2. To use the functions and data types of the Media Controller Client API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) applications), include the `<media_controller_client.h>` header file in your application:
```
Comment (mijin85cho) [inline docs/application/native/guides/multimedia/media-controller.md]: Please remove (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) applications).


```
2. To use the functions and data types of the Media Controller Client API, include the `<media_controller_client.h>` header file in your application:
```

File: docs/application/native/guides/multimedia/media-controller.md [hunk 3]
```diff
@@ -89,13 +89,19 @@ To enable your application to use the media controller functionality:
 
 - To use the media controller server:
 
-  1. To use the functions and data types of the Media Controller Server API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__SERVER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__SERVER__MODULE.html) applications), include the `<media_controller_server.h>` header file in your application:
+  1. The application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```
[the remark this answers is not in the document]
Comment (haejeongkim) [reply docs/application/native/guides/multimedia/media-controller.md]: Done

File: docs/application/native/guides/multimedia/media-controller.md [hunk 4]
```diff
@@ -89,13 +89,19 @@ To enable your application to use the media controller functionality:
...
+     ```
+     <privileges>
+        <privilege>http://tizen.org/privilege/mediacontroller.server</privilege>
+     </privileges>
+     ```
+  2. To use the functions and data types of the Media Controller Server API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__SERVER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__SERVER__MODULE.html) applications), include the `<media_controller_server.h>` header file in your application:
```
[the remark this answers is not in the document]
Comment (haejeongkim) [reply docs/application/native/guides/multimedia/media-controller.md]: Done

File: docs/application/native/guides/multimedia/media-controller.md (hunk 1 above)
Comment (haejeongkim) [reply docs/application/native/guides/multimedia/media-controller.md]: Done

File: docs/application/native/guides/multimedia/media-controller.md (hunk 2 above)
Comment (haejeongkim) [reply docs/application/native/guides/multimedia/media-controller.md]: Done

**Pull request merged**